### PR TITLE
fix fopen flags

### PR DESCRIPTION
**Pull request by SpacePossum**
| Q             | A
| ------------- | ---
| Branch?       | 2.7
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes 
| Fixed tickets | none
| License       | MIT
| Doc PR        | none

### CODE REVIEW

File: src/Symfony/Component/Process/Process.php [hunk 1]
```diff
@@ -1413,8 +1413,8 @@ private function resetProcessData()
         $this->exitcode = null;
         $this->fallbackStatus = array();
         $this->processInformation = null;
-        $this->stdout = fopen('php://temp/maxmemory:'.(1024 * 1024), 'wb+');
```
Comment (SpacePossum) [inline src/Symfony/Component/Process/Process.php]: `wb+` is not valid

File: src/Symfony/Component/Translation/Dumper/CsvFileDumper.php [hunk 2]
```diff
@@ -28,7 +28,7 @@ class CsvFileDumper extends FileDumper
      */
     public function format(MessageCatalogue $messages, $domain = 'messages')
     {
-        $handle = fopen('php://memory', 'rb+');
```
Comment (SpacePossum) [inline src/Symfony/Component/Translation/Dumper/CsvFileDumper.php]: `rb+` is not valid

**Comment by nicolas-grekas** [comment]
Hello, can you rebase+retarget on 2.8 please, and also keep only the fixed modes?
The `b` is already the default mode since PHP4.soemthing, it is not needed to make it explicit.

**Comment by SpacePossum** [comment]
sure thing @nicolas-grekas :+1:

**Review by nicolas-grekas — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Pull request merged**